scripts/SANS/isis_reduction_steps.py

```diff
@@ -3719,6 +3721,25 @@ def _readFrontRescaleShiftSetup(self, details, reducer):
                 else:
                     _issueWarning("Command: \"DET/" + details + "\" not valid. Expected format is /DET/RESCALE r")
 
+    def readFrontMergeRange(self, details, reducer):
+        """
+            Handle user commands of the type DET/OVERLAP [Q1 Q2] which are used to specify the range to merge
+
+            @param details: the contents of the line after DET/
+            @param reducer: the object that contains all the settings
+        """
+        values = details.split()
+        rAnds = reducer.instrument.getDetector('FRONT').mergeRange
+        rAnds.q_merge_range = False
+        if details.startswith('OVERLAP'):
+            if len(values) == 3:
+                rAnds.q_merge_range = True
+                rAnds.q_min = float(values[1])
+                rAnds.q_max = float(values[2])
+            else:
+                _issueWarning(
+                    "Command: \"DET/" + details + "\" not valid. Expected format is /DET/OVERLAP q1 q2")
+
```

Comment:
If `details` doesn't start with `OVERLAP` the command is silently ignored. Is it intentional?